example/src/plugins/job-ui-single/pages/CreateJobEntity.tsx

```diff
@@ -51,6 +54,12 @@ export const CreateJobEntity = (props: TCreateJobEntityProps) => {
   const [createdJobEntity, setCreatedJobEntity] = useState<TGenericObject>()
 
   const createJobEntity = (jobEntityFormData: TJob) => {
+    if (defaultJobOutputTarget) {
+      jobEntityFormData = {
+        ...jobEntityFormData,
+        outputTarget: defaultJobOutputTarget,
```

Comment:
If jobEntityFormData.outputTarget !== undefined, then it will still be overridden by the default. That seems odd.

Reply:
agree, should only be added if jobEntityFormData does not have an outputTarget attribute